docs/1.getting-started/2.installation.md

```diff
@@ -65,6 +65,12 @@ bunx nuxi@latest init <project-name>
 
 ::
 
+cd into your new project directory to begin using Nuxt.
+
+```bash
+cd <project-name>
+```
+
```

Comment:
I think instead of adding this we can remove the 'install the dependencies' section as this is now done automatically by nuxi.

Reply:
yes, in the end, nuxi is prompting me for `cd <project name>`